crypto/src/lib.rs

```diff
@@ -185,20 +183,16 @@ impl KeyPair {
 impl KeyPair {
     /// Digest function
     pub fn digest_function(&self) -> Algorithm {
-        self.private_key.digest_function()
+        self.private_key.algorithm()
```

Comment:
so is it `digest_function` or `algorithm`?

Reply:
Algorithm, thx